data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filter_attack_type.cfg

```diff
@@ -84,8 +86,46 @@
         {ASSERT ({VARIABLE_CONDITIONAL triggers equals 0})}
         {VARIABLE_OP triggers add 1}
     [/event]
+#enddef
+
+#####
+# API(s) being tested: [event][filter_attack]type=
+##
+# Actions:
+# Give Alice an ability that adds a damage special with addition of arcnae type to all of his weapons.
+# Give Bob resistance to -100% to arcane
+# Define events that use filter_attack matching Alice's arcane type.
+# Have Alice attack Bob.
+##
+# Expected end state:
+# An event triggers when Alice attacks during side 1's turn because type=arcane detected.
```

Comment:
An event triggers when Alice attacks because the attack is calculated using arcane damage

Reply:
yes